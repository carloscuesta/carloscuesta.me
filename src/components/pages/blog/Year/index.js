// @flow
import type { Element } from 'react'

import styles from './styles.module.css'

type Props = {
  year: string
}

const Year = (props: Props): Element<'div'> => (
  <div className='col-xs-12'>
    <h2 className={styles.year}>{props.year}</h2>
  </div>
)

export default Year
