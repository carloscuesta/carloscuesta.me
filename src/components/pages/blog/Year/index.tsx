import styles from './styles.module.css'

type Props = {
  year: string
}

const Year = (props: Props) => (
  <div className='col-xs-12'>
    <h2 className={styles.year}>{props.year}</h2>
  </div>
)

export default Year
