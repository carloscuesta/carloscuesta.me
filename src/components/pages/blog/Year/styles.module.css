.year {
  font-weight: 700;
  font-size: 2rem;
}
