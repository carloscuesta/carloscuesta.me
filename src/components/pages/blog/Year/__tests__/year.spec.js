import renderer from 'react-test-renderer'

import Year from '../index'

describe('Year', () => {
  it('should match Year component', () => {
    const wrapper = renderer.create(<Year year={2020} />)

    expect(wrapper).toMatchSnapshot()
  })
})
