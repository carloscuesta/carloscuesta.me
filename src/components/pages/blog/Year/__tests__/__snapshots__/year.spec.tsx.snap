// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Year should match Year component 1`] = `
<div
  className="col-xs-12"
>
  <h2
    className="year"
  >
    2020
  </h2>
</div>
`;
